src/GT.UI/Program.cs

```diff
@@ -1,9 +1,12 @@
 using GT.Core.Services.Impl;
 using GT.Core.Services.Interfaces;
 using GT.Data.Data.GTAppDb;
-using GT.Data.Data.GTAppDb.Entities;
```

Comment:
GTAppDb Entities not necessary in Program.cs?

Reply:
Fix in commit.